src/widget/container/mod.rs

```diff
@@ -114,7 +118,36 @@ where
         origin: Point,
         size: Size,
     ) -> Result<(), Display::Error> {
-        let mut y: i32 = 0;
+        let num_children = self.options.children.len() as u32;
+
+        if num_children == 0 {
+            return Ok(());
+        }
+
+        let total_children_height = self.content_size().height.unwrap_or(0);
+        let unused_height = size.height - total_children_height;
+
+        let (mut current_y, space) = match self.options.justification {
+            Justification::Start => (0, 0),
+            Justification::Center => (unused_height / 2, 0),
+            Justification::End => (unused_height, 0),
+            Justification::SpaceBetween => {
+                let space = if num_children > 1 {
+                    unused_height / (num_children - 1)
+                } else {
+                    0
+                };
+                (0, space)
+            }
```

Comment:
Because this is all done with integers, I think this could result in the final child not being flush with the bottom of the container, but I'm not too concerned for now.